persistence-jpa/javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/persistence/jpa/JpaReadSideImpl.java

```diff
@@ -127,14 +130,16 @@ private JpaReadSideHandler(
 
         @Override
         public Flow<Pair<Event, Offset>, Done, ?> handle() {
+            //This parameter shall be taken from configuration and will be global to this class (Not local to this method)
+	    boolean isBatchUpdateEnabled = true;
```

Comment:
Looks like there is some inconsistent indentation in these changes. Please make sure your editor matches Lagom's [standard indent configuraton](https://github.com/lagom/lagom/blob/master/.editorconfig). See http://editorconfig.org/ for links to plugins for various IDEs and editors that will configure this automatically.